cmd/run.go

```diff
@@ -94,7 +94,6 @@ func bindRunFlags(cmd *cobra.Command, config *app.Config) {
 	})
 }
 
-// TODO(dhruv): add more test only flags to this function.
```

Comment:
We only have one unsafe command for now. We can always add new unsafe commands, no need for a todo for that